experiments/superdarn_common_fields.py

```diff
@@ -64,3 +66,7 @@
 
 if opts.site_id in ["cly", "pgr"]:
     IS_REVERSE_RADAR = True
+
+def easy_scanbound(intt, beams):
```

Comment:
:+1: 
I'd like to suggest a comment here though: 
```suggestion
def easy_scanbound(intt, beams):
"""
Create integration time boundaries for the scan at the exact 
integration time (intt) boundaries. For new experiments, you 
may wish to ensure that your intt * len(beams) approaches a 
minute mark to reduce delay in waiting for the next scanbound.
"""
```